src/pybind/mgr/cephadm/serve.py

```diff
@@ -863,6 +864,7 @@ def _create_daemon(self,
                             'service_name': daemon_spec.service_name,
                             'ports': daemon_spec.ports,
                             'ip': daemon_spec.ip,
+                            'deployed_by': self.mgr.get_active_mgr_digests(),
```

Comment:
I wonder if instead of digests we should just the ceph version  string, e.g. ``16.1.0-917-g08ab4ae8``.  it's easier to get that the digests (we can drop get_active_mgr_digets() above) and it is also something a human can make sense of.

Reply:
I like the idea :+1:. I'll try it out and if there's no issues I'll swap over to that.